openslides/assignments/static/js/assignments/projector.js

```diff
@@ -22,11 +22,26 @@ angular.module('OpenSlidesApp.assignments.projector', ['OpenSlidesApp.assignment
         // Add it to the coresponding get_requirements method of the ProjectorElement
         // class.
         var id = $scope.element.id;
-        var poll = $scope.element.poll;
 
         // load assignemt object and related agenda item
         Assignment.find(id).then(function(assignment) {
             Assignment.loadRelations(assignment, 'agenda_item');
+            $scope.$watchGroup([assignment.assignment_related_users, $scope.element.poll.options],                                      function(newval, oldval) {
+                if (newval[0] != oldval[0]) {
```

Comment:
I cannot see where syntax is broken. I compared $watchgroup syntax with the [doc](https://docs.angularjs.org/api/ng/type/$rootScope.Scope), and can't see what's wrong